airflow/cli/commands/remote_commands/connection_command.py

```diff
@@ -108,18 +110,54 @@ def _connection_to_dict(conn: Connection) -> dict:
     }
 
 
-def create_default_connections(args):
-    db_create_default_connections()
+@provide_cli_api_client
+def create_default_connections(args, cli_api_client=NEW_CLI_API_CLIENT):
+    """Create default connections."""
+    cli_api_client.connections.create_defaults()
+    print("Default connections have been created.")
+
+
+def _response_to_connection(response: ConnectionResponse) -> Connection:
+    # TODO: Return is_encrypted from API to properly convert to Connection, otherwise it will be None
```

Comment:
What is `is_encrypted`? It's unclear from the context what exactly should be done as part of this TODO.

Reply:
`is_encrypted` is a column in the database and a parameter for `connection`. Normally, the direct database call returns this but we need to include this functionality if we want to bring back `is_encrypted`. This TODO is making that happen after this PR. I have updated the TODO :)